lib/distillery/releases/assembler.ex

```diff
@@ -655,7 +655,7 @@ defmodule Distillery.Releases.Assembler do
                {:ok, tokens, _} <- :erl_scan.string(String.to_charlist(templated)),
                {:ok, sys_config} <- :erl_parse.parse_term(tokens),
                :ok <- validate_sys_config(sys_config),
-               merged <- Mix.Config.merge(base_config, sys_config) do
+               merged <- Config.Reader.merge(base_config, sys_config) do
```

Comment:
Did you verify if these two `merge/2` functions work the same?

Reply:
Hello @CristhianRodriguezMolina, yes, the module `Mix.Config` is deprecated, and the `Config.Reader` module from the elixir standard library is recommended to work with configuration files. In this case, both functions do the same thing. More about the modules can be found here:
- Mix.Config: https://hexdocs.pm/mix/1.12.3/Mix.Config.html#merge/2
- Config.Reader: https://hexdocs.pm/elixir/1.12/Config.Reader.html#merge/2

Reply:
Understood, thank you.